### ISSUE TITLE
MicroMegaWatt Charger Bootup

### ISSUE THREAD

**Issue by faizanmir21** [EVerest/everest-demo#41]
Opening an issue for discussing the boot up issue for a PIONIX MicroMegaWatt Charger.

**Comment by faizanmir21** [comment]
Tried powering up the device using USB-C Power Delivery.

1. Conventional USB-C wall adapter.
2. Mac charger (rated 100W) with USB PD output specs as: (20.5V 4.7A), (15V 3A), (9V 3A), (5.2V 3A)

The tech sheet on the PIONIX website states the required input as: 
–USB-C
–Any powerbank capable of 12V >3A via Power Delivery protocol

The input requirements on the charging devices are: 12V 2A

**Comment by faizanmir21** [comment]
Posted the issue on #everest zulip chat, waiting for response.

**Comment by faizanmir21** [comment]
Here is what I found on the internet. 

USB-C PD 1.0 supports 12V output however it has been replaced by PD 2.0 and 3.0 which is used by most new electronics and doesn't output 12V.
![Screenshot 2024-05-15 at 12 16 24 PM](https://github.com/EVerest/everest-demo/assets/92818244/f158a4d3-d4dd-4599-9026-4e2d9d4e3c04)

**Comment by shankari** [comment]
so the mac adapter, for example, would be USB PD 2/profile 3. Can you ask around at MBAP whether anybody has an older smartphone charger? I am not sure if people know the version of the adapter they have, though. I certainly don't

**Comment by faizanmir21** [comment]
So, I used a different charger and was able to boot up the device. I kept the device plugged in for a long time and it was able to boot up since it was the first boot and now it works as expected.
The PCB inside the charger case has an led which lights up when plugged in, it is hard to see that in a bright room but I plugged it in a dark room, noticed the led light up and kept the device plugged in for a while. Checked back in it was running.

**Comment by faizanmir21** [comment]
Couldn't access it through terminal because it asked for the password. But was able to see the log files from the browser.
<img width="1260" alt="Screenshot 2024-05-20 at 1 03 39 PM" src="https://github.com/EVerest/everest-demo/assets/92818244/9d54bd4c-b5d7-4211-a0a7-08d3bb05fea1">

**Comment by shankari** [comment]
doesn't the documentation that came with the box include the password? We will need ssh access to update the version of EVerest running on it. I'm also surprised to see logs saying `EVConnected` and `EIM Authorization Received`. Did you actually plug the car in?

**Comment by faizanmir21** [comment]
No, there is no documentation in the box. I can try posting in tulip chat if there is a generic one.
I did not plug it in the car, not sure why its saying that.

**Comment by shankari** [comment]
That might be another question to ask on the chat

**Comment by faizanmir21** [comment]
Was able to ssh into the MMW charger.
Username: everest
Password: everest
